css/style.css

```diff
@@ -1,4 +1,503 @@
+  .menubar__link {
+    display: none;
+  }
+  .tesla-bgd, .tesla-car {
+    height: auto;
+    width: 100%;
+    max-width: 1023px;
+  }
+  .tesla-bgd{
+    top: -35px;
+    left: 0;
+  }
+  .tesla-car {
+    padding: 54px 0 20px 4px;
+  }
+  .content__data-car {
+    padding: 0 calc(321px - 2 * (321px - (321 * 100% / 1440))) 50px;
+    flex-direction: column;
+  }
+  .content__facts {
+    padding-bottom: 65px;
+  }
+  .data__speed-block {
+    padding-right: 144px;
+  }
+  .data__speed-icon {
+    width: 65px;
+    height: 50px;
+  }
+  .content__road-icon {
+    width: 43px;
+    height: 71px;
+  }
+  .data__dimention {
+    align-items: center;
+  }
+  .data__figures {
+    font-size: 50px;
+  }
+  .data__specifications {
+    display: block;
+  }
+  .testdrive__img {
+    display: none;
+  }
+  .testdrive__text {
+    position: relative;
+    font-size: 16px;
+    color:  #e51937;
+  }
+  .testdrive__text:before {
+    content: '';
+    background-image: url("/img/png/steering-wheel-red.png");
+    display: block;
+    position: absolute;
+    width: 32px;
+    height: 32px;
+    left: -45px;
+    top: -8px;
+  }
+  .order-btn {
+    width: 221px;
+    height: 55px;
+    background-color: #e51937;
+    color: #fff;
+  }
+}
+@media (min-width:768px) and (max-width:899px) {
+  .header {
+    padding: 25px 24px 96px 0;
+  }
+  .header__logo-img{
+    padding-left: 15px;
+    width: 167px;
+    height: 50px;
+  }
+  .menubar__link {
+    display: none;
+  }
+  .button {
+    width: 33px;
+    height: 20px;
+  }
+  .second {
+    top: 9px;
+  }
+  .third {
+    top: 18px;
+  }
+  .tesla-bgd{
+    top: -22px;
+    width: 830px;
+    height: 271px;
+    padding-left: 35px;
+  }
+  .tesla-car {
+    width: 701px;
+    height: 332px;
+    padding: 84px 0 20px 33px;
+  }
+  .content__data-car {
+    flex-direction: column;
+    padding: 11px 0 55px 58px;
+  }
+  .content__facts {
+    padding: 0px 0px 54px 0px;
+  }
+  .data__speed-block {
+    padding-right: 92px;
+  }
+  .data__speed-icon {
+    width: 78px;
+    height: 62px;
+    padding: 7px 14px 0 0;
+  }
+  .content__road-icon {
+    width: 48px;
+    height: 77px;
+  }
+  .data__dimention {
+    align-items: center;
+  }
+  .data__figures {
+    font-size: 51px;
+    padding-right: 6px;
+  }
+  .data__specifications {
+    display: block;
+  }
+  .testdrive__img {
+    display: none;
+  }
+  .testdrive__text {
+    position: relative;
+    font-size: 16px;
+    color:  #e51937;
+  }
+  .testdrive__text:before {
+    content: '';
+    background-image: url("/img/png/steering-wheel-red.png");
+    display: block;
+    position: absolute;
+    width: 32px;
+    height: 32px;
+    left: -45px;
+    top: -8px;
+  }
+  .order-btn {
+    width: 221px;
+    height: 55px;
+    background-color: #e51937;
+    color: #fff;
+    font-size: 16px;
+  }
+  .arrow {
+    padding-left: 2px;
+  }
+  
+}
+ @media (min-width:568) and (max-width: 768px) {
+
+ }
+ @media (min-width:320) and (max-width: 568px) {
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5652830)